sql/4_3_0-to-4_3_1_upgrade.sql

```diff
@@ -127,3 +127,9 @@ ALTER TABLE patient_data ADD COLUMN billing_note text NOT NULL default '';
 UPDATE `patient_data` SET `billing_note` = `genericval2` WHERE `genericname2` = 'Billing';
 UPDATE `patient_data` SET `genericval2` = '', `genericname2` = '' WHERE `genericname2` = 'Billing';
 #EndIf
+
+#IfMissingColumn lang_languages lang_is_rtl
+ALTER TABLE `lang_languages` ADD COLUMN `lang_is_rtl` TINYINT DEFAULT 0;
+UPDATE `lang_languages` SET `lang_is_rtl`=1 WHERE `lang_code` in ('he','ar');
```

Comment:
Note that the lang_code will likely change in the future. Will be better to instead use the lang_description setting.